src/dateTime.js

```diff
@@ -1,43 +1,37 @@
 import { GraphQLScalarType, GraphQLError, Kind  } from "graphql"
 import * as Joi from "@hapi/joi"
 
+const isValidDateTime = new RegExp(/^\d{4}-\d\d-\d\dT\d\d:\d\d:\d\d(\.\d+)?(([+-]\d\d:\d\d)|Z)?$/i);
+
 const validate = value => {
-  Joi.assert(value, Joi.date().iso(), new TypeError(`Value is not a valid Date: ${value}`))
-  return value
+  Joi.assert(new Date(Date.parse(value)), Joi.date().iso(), new TypeError(`Value is not a valid Date: ${value}`))
```

Comment:
Why do you need `Date.parse` ?

Reply:
Initially the argument `value` was a string transformed by `new Date(Date.parse(string))` where validate is called.